internal/pkg/manifest/manifest.go

```diff
@@ -73,27 +74,37 @@ func LoadManifest(fs filesystem.Fs) (*Manifest, error) {
 		return nil, err
 	}
 
-	// Connect together all manifest records
+	// Read all manifest records
 	for _, branch := range m.Content.Branches {
-		if err := m.PersistRecord(branch); err != nil {
+		if err := m.trackRecord(branch); err != nil {
 			return nil, err
 		}
 	}
 	for _, config := range m.Content.Configs {
-		if err := m.PersistRecord(config.ConfigManifest); err != nil {
+		if err := m.trackRecord(config.ConfigManifest); err != nil {
 			return nil, err
 		}
 		for _, row := range config.Rows {
 			row.BranchId = config.BranchId
 			row.ComponentId = config.ComponentId
 			row.ConfigId = config.Id
-			if err := m.PersistRecord(row); err != nil {
+			if err := m.trackRecord(row); err != nil {
 				return nil, err
 			}
 		}
 	}
 
+	// Connect records together and resolve parent path
```

Comment:
Teraz riesim podporu pridania `variables` a ta nam to komplikuje.
Kedze:
- `variables` su reprezentovane tiez ako `config`, pre komponentu `keboola.variables`
- je to takto, lebo je to POC :/ a nema to vlastne API
- `variables` maju byt ulozene vramci konfiguracie, ktora ich pouziva, je tam vztah `1:1`
- teda takto nam vznikla zavislost `config <-> config` ... 
- a na adresarovej urovni je pre `variables` config, rodicom `config`, ktory ich pouziva.
- ^^^^ je to zrozumitelne? ak nie tak zavolaj .... 

No a teda uz nam nestaci spracovat najprv `branches` a potom `configs`, aby sme v case volania `PersistRecorrd` mali istotu, ze `parent` uz bol nacitany.

Reply:
Preto je nacitanie rozdelene na 2 casti:
- naprv sa vola `trackRecord`, ktory iba ulozi referenciu na zaznam
- a ked mame vsetky zaznami nacitane, tak sa vola `PersistRecord`, ktory riesi naming a nastavi `parentPath`.